src/tracks/SampleTrack.cpp

```diff
@@ -321,6 +336,36 @@ void SampleTCO::loadSettings( const QDomElement & _this )
 	if (_this.hasAttribute("sample_rate")) {
 		m_sampleBuffer->setSampleRate(_this.attribute("sample_rate").toInt());
 	}
+
+	if (_this.hasAttribute("leftP1x") && _this.hasAttribute("leftP1y") &&
+	    _this.hasAttribute("leftP2x") && _this.hasAttribute("leftP2y"))
```

Comment:
I think this looks and reads better.
```suggestion
	if (_this.hasAttribute("leftP1x") && _this.hasAttribute("leftP1y")
	    && _this.hasAttribute("leftP2x") && _this.hasAttribute("leftP2y"))
```

Reply:
Agreed. Could you please mark the one for "right" too, so i can batch commit?